package.json

```diff
@@ -3,18 +3,50 @@
   "version": "1.0.0",
   "description": "gtr-cof",
   "scripts": {
+    "clean": "rm -rf docs/scripts",
+    "build:dist": "NODE_ENV=production webpack",
+    "lint": "tslint -c tslint.json --project tsconfig.json",
     "lite": "lite-server --baseDir=\"docs\" --port 10001",
     "start": "npm run lite",
-    "dev": "concurrently --raw --kill-others 'npm start' 'tsc -w'"
+    "dev": "concurrently --raw --kill-others 'npm run start' 'npm run type-check -- -w' 'npm run watch' ",
+    "precommit": "lint-staged",
+    "prepush": "npm run type-check",
+    "prettier": "npm run prettier",
+    "type-check": "tsc --noEmit --pretty",
```

Comment:
Since webpack now handles compilation output, `tsc` is only used for type-checking. Note that webpack reads in the `tsconfig.json` file for its settings.